src/plugins/action/action.h

```diff
@@ -3,7 +3,7 @@
 #include <functional>
 #include <memory>
 
-#include <dronecode_sdk/plugin_base.h>
+#include "plugin_base.h"
```

Comment:
Will this work when installed?

Reply:
I believe that `<>` vs `"` is merely a convention. Most compilers will look for system includes first when you use the angle brackets, but I believe it still doesn't mean "all".

I think it works when building/running the examples in the CI, doesn't it? Do you mean if that's installed on the system?

Reply:
Yes, does it work when installed in the system?

Reply:
I added a Jenkins stage for that, we'll see :D.

Reply:
So yes, that's working